### PR TITLE
feat/[sfa-16]: remove liquidity nft

### PR DESCRIPTION
**Pull request by freitasgouvea**

### CODE REVIEW

**Review by luanpontolio — COMMENTED** [review]
The PR has a WIP in the name.

There are some comments and `console.logs` do you think it is better to maintain this?

File: apps/web/src/components/SearchModal/CommonBases.tsx [hunk 1]
```diff
@@ -72,8 +71,11 @@ export default function CommonBases({
   useMemo(async () => {
       if (collections?.length && chainId) {
         const newCollection = collections.map(async (collection) => {
-          const contract = getErc721Contract(collection.id, data )
-          const symbol = await contract.symbol()
+          const erc721Contract = getErc721Contract(collection.id, data )
+          const symbol = await erc721Contract.symbol()
+
+          const factoryContract = getIUniswapV2FactoryContract(FACTORY_ADDRESS[chainId], data )
```
Comment (luanpontolio) [inline apps/web/src/components/SearchModal/CommonBases.tsx:77]: Maybe we can create a hook useFactoryContract(). What do you think?

File: apps/web/src/components/SeeNftsModal/SeeNftsModal.tsx [hunk 2]
```diff
@@ -1,53 +1,60 @@
...
 import styled from 'styled-components'
 import EthIcon from './Svg/IconsSeeNftsModal'
+import { Item } from 'types/Item'
 
-const Title = () => {
+const Title = (collectionName: string, collectionImage?: string) => {
   return (
     <Header>
-      <img src="https://testnets.opensea.io/static/images/placeholder.png" alt="" />
+      <img src={collectionImage ?? 'https://testnets.opensea.io/static/images/placeholder.png'} alt="" />
```
Comment (luanpontolio) [inline apps/web/src/components/SeeNftsModal/SeeNftsModal.tsx:9]: Maybe it is better to add the `placeholder.png`  in our project

File: apps/web/src/components/SeeNftsModal/SeeNftsModal.tsx [hunk 3]
```diff
@@ -1,53 +1,60 @@
...
-      <Action>
+      {/* <Action>
         <Button as="button" scale="sm" variant="secondary" width="100%" onClick={onDismiss}>
           Cancel
         </Button>
         <Button as="button" scale="sm" variant="primary" width="100%">
           Confirm
         </Button>
-      </Action>
+      </Action> */}
```
Comment (arthurmr96) [inline apps/web/src/components/SeeNftsModal/SeeNftsModal.tsx:57]: Remove the comment if it's not necessary anymore

**Review by arthurmr96 — CHANGES_REQUESTED** [review]

**Review by arthurmr96 — APPROVED** [review]

**Pull request merged**